# Use intel specific default 8bytle long integer and real swithces
# August 20th 2018 by B. Hajgato (Free University Brussels - VUB)
--- Assimulo-2.9/setup.py.orig	2018-08-13 15:41:24.186453384 +0200
+++ Assimulo-2.9/setup.py	2018-08-13 15:43:56.579604164 +0200
@@ -458,7 +458,7 @@
         src=['assimulo'+os.sep+'thirdparty'+os.sep+'odassl'+os.sep+code for code in odassl_list]
         config.add_extension('assimulo.lib.odassl', sources= src, include_dirs=[np.get_include()],**extraargs)
     
-        dasp3_f77_compile_flags = ["-fdefault-double-8","-fdefault-real-8"]
+        dasp3_f77_compile_flags = ["-i8","-r8"]
         dasp3_f77_compile_flags += extra_compile_flags
         
         if StrictVersion(np.version.version) > StrictVersion("1.6.1"): #NOTE, THERE IS A PROBLEM WITH PASSING F77 COMPILER ARGS FOR NUMPY LESS THAN 1.6.1, DISABLE FOR NOW
